From e7711e5cfbe6e5633831f2f82c99e016b086b5bb Mon Sep 17 00:00:00 2001
From: Demetris Ierokipides <ierokipides.dem@gmail.com>
Date: Wed, 1 Feb 2023 00:02:23 +0200
Subject: [PATCH] LOCAL: configs: meson64: prevent stdout/stderr on 
 videoconsole

Several devices have CONFIG_DM_VIDEO enabled which causes stdout/stderr
to appear on videoconsole, so remove videoconsole from STDOUT so that
early u-boot boot remains silent unless using the uart/serial console.
- Rebase for v2023.01 by Demetris
---
 include/configs/meson64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/meson64.h b/include/configs/meson64.h
index 726f33c2..f97f83d8 100644
--- a/include/configs/meson64.h
+++ b/include/configs/meson64.h
@@ -25,7 +25,7 @@
 
 /* For splashscreen */
 #ifdef CONFIG_VIDEO
-#define STDOUT_CFG "vidconsole,serial"
+#define STDOUT_CFG "serial"
 #else
 #define STDOUT_CFG "serial"
 #endif
-- 
2.34.1

